pyccel/ast/variable.py

```diff
@@ -551,6 +551,15 @@ def name(self):
     def __str__(self):
         return """.""".join(str(n) for n in self.name)
 
+    def __eq__(self, other):
+        return str(self) == other
+
+    def __ne__(self, other):
+        return str(self) != other
```

Comment:
Wouldn't it be safer to write `str(other)`? Or is that a use case that should never happen?

Reply:
I agree

Reply:
Fixed 80b6fe9f